shell/src/main/java/org/apache/accumulo/shell/commands/QuotedStringTokenizer.java

```diff
@@ -72,7 +72,7 @@ private void createTokens() throws BadArgumentException, UnsupportedEncodingExce
         inEscapeSequence = false;
         if (ch == 'x') {
           hexChars = "";
-        } else if (ch == ' ' || ch == '\'' || ch == '"' || ch == '\\') {
+        } else if (ch == ' ' || ch == '\'' || ch == '"' || ch == '\\' || ch == ':') {
```

Comment:
Not sure if this change is necessary for regards to this PR. If you think it still is then some of the comments should be updated to include the ':' being added.

Reply:
We don't need this. We can revert this back. Let me do that